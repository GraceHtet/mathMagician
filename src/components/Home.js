import homeStyle from './styles/home.module.css';

const Home = () => (
  <>
    <h2>Welcome to our page</h2>
    <div className={homeStyle.info}>
      <p>
        A calculation is a deliberate mathematical process that transforms one
        or more inputs into one or more outputs or results. The term is used in
        a variety of senses, from the very definite arithmetical calculation of
        using an algorithm.
      </p>
      <p>
        Math magic tricks can liven up any math class and create a sense of
        wonder and curiosity about math. Not only that, math magic creates a new
        context for algebraic reasoning as students go beyond What is the
        answer? to explore What is the trick?
      </p>
    </div>
  </>
);

export default Home;
