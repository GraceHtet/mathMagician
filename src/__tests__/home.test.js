import * as React from 'react';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

describe('Home', () => {
  test('Home Page Title match SnapShot', () => {
    render(<Home />);
    expect(screen.getByText(/Welcome to our page/i)).toMatchSnapshot();
  });

  test('Home Text match SnapShot', () => {
    render(<Home />);
    expect(screen.getByText(/A calculation is/i)).toMatchSnapshot();
  });
});
