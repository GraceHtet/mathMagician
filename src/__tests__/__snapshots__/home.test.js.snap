// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home Page Title match SnapShot 1`] = `
<h2>
  Welcome to our page
</h2>
`;

exports[`Home Home Text match SnapShot 1`] = `
<p>
  A calculation is a deliberate mathematical process that transforms one or more inputs into one or more outputs or results. The term is used in a variety of senses, from the very definite arithmetical calculation of using an algorithm.
</p>
`;
